builtin/game/chatcommands.lua

```diff
@@ -71,7 +71,7 @@ end
 --
 core.register_chatcommand("me", {
 	params = "<action>",
-	description = "chat action (eg. /me orders a pizza)",
+	description = "Display chat action (eg. /me orders a pizza)",
```

Comment:
Ok, i was just not aware of popular use of 'chat action', so that's fine. The example you suggest would be good.